.github/workflows/abi_go_bindings_checker.yml

```diff
@@ -0,0 +1,48 @@
+name: ABI Go Bindings Checker
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+
+env:
+  GO_VERSION: "1.20.7"
+
+jobs:
+  abi_binding:
+    name: abi_binding
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout go-ethereum repository
+        uses: actions/checkout@v4
+        with:
+          repository: ethereum/go-ethereum
+          ref: v1.13.1
```

Comment:
Do we need to checkout Geth? I thought we were using `subnet-evm` `abigen`

Reply:
yes, removed